network/tor/tor/src/main/java/bisq/network/tor/process/control_port/ControlPortFilePoller.java

```diff
@@ -45,26 +48,94 @@ private void startPoller() {
         Thread thread = new Thread(() -> {
             ThreadName.setName("ControlPortFilePoller.startPoller");
             try {
-                while (true) {
+                int attemptLimit = 300; // e.g., try for up to 30 seconds (300 * 100ms interval)
```

Comment:
Maybe for consistency rename to MAX_ATTEMPTS and make it final static at class level